examples/imagenet/train_imagenet.py

```diff
@@ -153,7 +162,10 @@ def main():
     # using InfiniBand and MultiprocessIterator. This is because processes
     # often crash when calling fork if they are using Infiniband.
     # (c.f., https://www.open-mpi.org/faq/?category=tuning#fork-warning )
-    multiprocessing.set_start_method('forkserver')
+    try:
+        multiprocessing.set_start_method('forkserver')
+    except AttributeError:
```

Comment:
I think this try-except statement is not a good idea because of the following reasons:
1.  Without `multiprocessing.set_start_method`, this example will always crash when using multiple nodes and IB, so this change will allow Python 2 users to encounter crash problems.
2. We want to keep our examples as simple as possible, as examples are for new users to learn how to use ChainerMN.

How about checking the Python version at the beginning, and exclude Python versions that do not have `multiprocessing.set_start_method` ?